Parsers/ASimFileEvent/Parsers/vimFileEventSentinelOne.yaml

```diff
@@ -0,0 +1,109 @@
+Parser:
+  Title: File Event Parser for SentinelOne
+  Version: '0.1.1'
+  LastUpdated: July 24, 2023
+Product:
+  Name: SentinelOne
+Normalization:
+  Schema: FileEvent
+  Version: '0.2.1'
+References:
+- Title: ASIM File Event Schema
+  Link: https://aka.ms/ASimFileEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: SentinelOne Documentation
+- Link: https://<SOneInstanceDomain>.sentinelone.net/api-doc/overview
+Description: |
+  This ASIM parser supports normalizing SentinelOne logs to the ASIM File Event normalized schema. SentinelOne events are captured through SentinelOne data connector which ingests SentinelOne server objects such as Threats, Agents, Applications, Activities, Policies, Groups, and more events into Microsoft Sentinel through the REST API.
+ParserName: vimFileEventSentinlOne
+EquivalentBuiltInParser: _Im_FileEvent_SentinelOne
+ParserQuery: |
+  let ASIM_GetWindowsFilenamePart = (path: string) { tostring(split(path, @'\')[-1]) };
+  let ASIM_GetLinuxFilenamePart = (path: string) { tostring(split(path, @'/')[-1]) };
```

Comment:
we use this name for out of box functions. Can you use 'GetLinuxFilenamePart' and 'GetWindowsFilenamePart'?

Reply:
Done.